entity-framework/toc.yml

```diff
@@ -223,8 +204,66 @@
         href: core/saving/disconnected-entities.md
       - name: Explicit values for generated properties
         href: core/saving/explicit-values-generated-properties.md
-    - name: Supported .NET implementations
-      href: core/platforms/index.md
+
+    #- name: Change tracking
+    #  items:
+    #  - name: Overview <<<--- No Page
+    #  - name: Property changes <<<--- No Page
+    #  - name: Relationship changes <<<--- No Page
+    #  - name: Find and local queries <<<--- No Page
+    #  - name: Accessing entity state <<<--- No Page
+    #  - name: Notification entities <<<--- No Page
+    #  - name: Change tracking proxies <<<--- No Page
+
+    - name: Logging, events, and diagnostics
+      items:
+      - name: Overview <<<--- Moved from events/index.md
+        href: core/logging-events-diagnostics/index.md
+      - name: Logging <<<--- former page with no notes on where to move
+        href: core/logging-events-diagnostics/logging.md
+      - name: Simple logging <<<--- Moved from events/simple-logging.md
+        href: core/logging-events-diagnostics/simple-logging.md
+      #- name: Microsoft.Extensions.Logging <<<--- No Page
+      #- name: ToQueryString <<<--- No Page
+      #- name: Events <<<--- No Page
+      #- name: Interceptors <<<--- No Page
+      #- name: Diagnostic listeners <<<--- No Page
+      #- name: Debug views <<<--- No Page
+      #- name: Event counters <<<--- No Page
+
+    - name: Testing
+      items:
+      - name: Testing code that uses EF Core
+        href: core/testing/index.md
+      - name: EF Core testing sample
+        href: core/testing/testing-sample.md
+      - name: Sharing databases between tests
+        href: core/testing/sharing-databases.md
+      - name: Test with SQLite
+        href: core/testing/sqlite.md
+      - name: Test with InMemory
+        href: core/testing/in-memory.md
+
+    - name: Miscellaneous
+      items:
+      - name: Supported .NET implementations
+        href: core/miscellaneous/platforms.md
+      - name: Asynchronous programming
+        href: core/miscellaneous/async.md
+      - name: Nullable reference types
+        href: core/miscellaneous/nullable-reference-types.md
+      #- name: Using dependency injection <<<--- No Page
+      - name: Collations and case sensitivity
+        href: core/miscellaneous/collations-and-case-sensitivity.md
+      - name: Connection resiliency
+        href: core/miscellaneous/connection-resiliency.md
+      - name: Configuration and initialization <<<--- formerly Configure a DbContext
```

Comment:
Move to top level, "DbContext configuration and initialization"

See [_DbContext Configuration and Initialization_]() for more information.